.changeset/tasty-turkeys-brake.md

```diff
@@ -0,0 +1,61 @@
+---
+'@roadiehq/backstage-plugin-argo-cd-backend': minor
+---
+
+## 🚀 New Feature:
+
+You can now use azure active directory (Microsoft Entra ID) to authentication to Argo CD instances!
+
+**Login Priorities** (We will prioritize logins as follows):
+
+1. Instance Level Tokens
+
+   ```json
+       argocd:
+           appLocatorMethods:
+               - type: 'config'
```

Comment:
There's no need to add quotes around `config`. It's a string just like all the other values.